rootfs-builder/rootfs.sh

```diff
@@ -92,9 +92,19 @@ check_function_exist() {
 generate_dockerfile() {
 	dir="$1"
 
+	case "$(arch)" in
```

Comment:
Shame there is not some nice automatic mapping for all arch's :-(
/cc @harche in case we want/need to add a Power ref in here as well

Reply:
Couldn't we just remove the case and set:

```
arch=$(go env GOARCH)
```

... as I think that will dtrt for the values below without the need to remap?

Reply:
The only con for use `go env GOARCH` is that will require go when we want is to minimize the depedencies to only docker when using dockerfiles. Both cases work for me.

Reply:
That's a fair point. Maybe we should use a docker image (with go installed) to... generate the Dockerfiles?

/me runs...

Reply:
ha, go is required to install go  😆